tooling-docker/src/test/java/org/jboss/eap/qe/ts/common/docker/DockerTest.java

```diff
@@ -0,0 +1,42 @@
+package org.jboss.eap.qe.ts.common.docker;
+
+import io.restassured.RestAssured;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import static org.hamcrest.Matchers.containsString;
+
+public class DockerTest {
+
+    @ClassRule
+    public static Docker wildflyContainer = new Docker.Builder("wildfly", "jboss/wildfly:18.0.0.Final")
```

Comment:
Put it to `DockerContainers` and make version configurable via argument ?

Reply:
Will anyone use Wildfly container? I used in test just because everyone knows it and it can be easily tested.

Reply:
it's not needed now, maybe in the future, we can skip that for now